src/clj/clojurians_log/db/queries.clj

```diff
@@ -34,6 +53,10 @@
             db
             chan-name
             day)
+
+       ;; Remove all thread messages except for the thread parent
+       (remove #(if-let [thread-ts (:message/thread-ts %)]
+                  (not= thread-ts (:message/ts %))))
```

Comment:
I would simplify this to `(not= (:messsage/thread-ts %) (:message/ts %))`, if thread-ts is not present than this will evaluate to `(not= nil ts)`, which is also false.

Reply:
That's how I had it originally. However, I found that this *always* moved the parent message itself. Turns out, the parent message itself doesn't have a "thread-ts" field. So for parent messages `(:message/thread-ts %)` is always nil, but `(:message/ts %)` was always *something*, thus causing the parent message's removal.

Reply:
wow, that's interesting, and pretty annoying, I thought we could rely on `:thread-ts` == `:ts` to identify a thread parent, but guess not. (in hindsight this makes sense, you can't know beforehand if a message will get replies or not, even old messages could still become thread parents).

Maybe comment this more explicitly as 

```
;; thread parents do not have a :thread-ts value themselves 
```